cfgov/unprocessed/js/modules/util/arrays.js

```diff
@@ -0,0 +1,28 @@
+function forEach( array, callback, scope ) {
```

Comment:
Ideally this should move to a polyfill in `js/modules/polyfills/` to cover IE8 and the code that calls this should just use https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach